src/main/java/org/jenkinsci/plugins/jenkinsreviewbot/ReviewboardPollingBuilder.java

```diff
@@ -195,6 +190,14 @@ public void perform(Run<?, ?> run, FilePath workspace, Launcher launcher, TaskLi
       listener.getLogger().println("Problem starting the Jenkins job!");
       e.printStackTrace(listener.getLogger());
     }
+    // Write the reviews to the file, they will be processedReviews for the next run
+    listener.getLogger().println("Writing reviews to file...");
+    try {
+      writeReviews(file, reviews);
```

Comment:
Same thing can still happen here - If the job doesn't start, an exception is caught and logged and the review is still marked as processed.

We could potentially create a set of already processed reviews (reviews - unprocessedReviews) and append a review after line 187 when the job has successfully been scheduled.